htdocs/install/mysql/migration/16.0.0-17.0.0.sql

```diff
@@ -389,3 +389,4 @@ ALTER TABLE llx_opensurvey_comments ADD COLUMN date_creation datetime NULL;
 
 ALTER TABLE llx_c_tva ADD COLUMN use_default tinyint DEFAULT 0;
 
+ALTER TABLE llx_socpeople ADD COLUMN fk_parent integer DEFAULT 0 NOT NULL AFTER fk_stcommcontact;
```

Comment:
Why not instead :
ALTER TABLE llx_socpeople ADD COLUMN fk_parent integer NULL AFTER fk_stcommcontact;
?
Value fk_parent will be filled only when a parent exists and if we add an index on field, it will consume only for fk_parent defined.

Reply:
Ok i enderstand it's done